src/components/player/download-control.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react'
+import {FunctionComponent} from 'react'
+import axios from 'utils/configured-axios'
+import {track} from 'utils/analytics'
+import Tippy from '@tippyjs/react'
+
+type DownloadButtonProps = {
+  lesson: any
+}
+
+type DownloadControlProps = {
+  lesson: any
+  key: string
+  order: number
+}
+
+const DownloadButton: FunctionComponent<DownloadButtonProps> = ({lesson}) => {
```

Comment:
@DrShpongle download control is less critical, but same idea, anything in controlbar will utilize cueplayer patterns for styling and layout so that it is portable with the player and not assume tailwind is present